ccmlib/node.py

```diff
@@ -920,7 +920,12 @@ def get_sstables(self, keyspace, column_family):
                 files.remove(f)
         return files
 
-    def stress(self, stress_options=[], **kwargs):
+    def stress(self, stress_options=None, **kwargs):
+        if stress_options is None:
+            stress_options = []
+        else:
+            stress_options = stress_options[:]
```

Comment:
To use a copy of a passed list rather than the list itself, since the method is doing in-place modification which can cause side effects in the calling code. I found this by accident when running node.stress multiple times with a passed in list (same list in the node.stress call multiple times) triggering syntax errors because -node was appended multiple times.